### PR TITLE
policy

### PR DESCRIPTION
**Pull request by flavio**
Add the initial policy.

Important: the policy annotation will fail, because the standard kwctl cannot handle `wasi` policies yet

Fixes https://github.com/kubewarden/kubewarden-controller/issues/472

### CODE REVIEW

File: README.md
Comment (jvanz) [inline README.md:2]: Are you referring to the TODO inside the code?

File: kw_sdk.go
Comment (jvanz) [inline kw_sdk.go:22]: I see the benefits of using a contextual name for a type. Thus, why not using this type inside the `ValidationResponse` and `SettingsValidationResponse` types in the `Message` field?

File: metadata.yml
Comment (jvanz) [inline metadata.yml:18]: Diverge from the `oci-target` used in the release workflow

File: settings_test.go
Comment (jvanz) [inline settings_test.go:83]: Go `testing.T`  has [function](https://pkg.go.dev/testing@go1.20.5#T.Run) to allow us to define subtests. This way, in the tests output is possible to know which subtest failed by name. I believe this is nice to do here.

File: validate_test.go
Comment (jvanz) [inline validate_test.go:109]: The same suggestion that I did in the settings validation tests. Maybe we should use `testing.T.Run` function here.

**Review by jvanz — CHANGES_REQUESTED** [review]

File: README.md
Comment (flavio) [reply README.md:2]: It's meant to be the link to the https://deploy-preview-195--silly-bunny-8cedd0.netlify.app/writing-policies/wasi section. I've updated the README to include the final URL

File: README.md
Comment (flavio) [reply README.md:47]: The majority of our policies, especially the trivial ones, omit the full Kubernetes CR and focus only on the settings of themselves. I prefer this approach when the policy is trivial, because it keeps everything more concise

File: LICENSE
Comment (flavio) [reply LICENSE:1]: done, thanks

File: .github/workflows/release.yml
Comment (flavio) [reply .github/workflows/release.yml:58]: fixed

File: metadata.yml
Comment (flavio) [reply metadata.yml:18]: thanks, fixed

File: settings_test.go
Comment (flavio) [reply settings_test.go:83]: you're right, thanks for the hint!

File: validate_test.go
Comment (flavio) [reply validate_test.go:109]: done

File: kw_sdk.go
Comment (flavio) [reply kw_sdk.go:22]: BTW, this is copy and pasted with some minor changes from the official go sdk.

I don't know why we didn't do that. I think the reasoning was that regular users create these objects with the helper function we provide (like `RejectRequest`); these functions hint good practices by using this type.
When dealing with "low level" objects, we just use vanilla types for simplicity.

Is it worth to change that? I fear this would cause some renaming, otherwise we would end up with something like that:

```golang
type ValidationResponse {
  // other fields
  Message *Message
}
```

which is a bit weird IMHO

Comment (jvanz) [reply kw_sdk.go:22]: Okay. Fine by me. Let's keep it as it is now.

**Comment by jvanz** [comment]
@flavio I cannot merge this PR. I do not have write access.

**Pull request merged**
